tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -320,14 +325,14 @@ def test_max_partitions(self):
             self.redpanda,
             "ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
             (workload, validator),
-            num_workers=10,
+            num_workers=self.CLUSTER_NODES - 1,
```

Comment:
Does OMB make use of uneven nodes in "ensemble" mode?

Reply:
Yes, it assigns any "odd" worker to the consumer side, which I guess is good for us since we use 3:1 and 2:1 ratios, so maybe consumers are more likely to be overloaded (not in the many-partitions case though I guess due to the issue with high CPU in the drain loop there). 

```
	int numberOfProducerWorkers = extraConsumerWorkers ? (workers.size() + 2) / 3 : workers.size() / 2;
	List<List<String>> partitions = Lists.partition(Lists.reverse(workers), workers.size() - numberOfProducerWorkers);
	this.producerWorkers = partitions.get(1);
	this.consumerWorkers = partitions.get(0);
```

(AFAIK we don't set `extraConsumerWorkers`)